docs/sources/release-notes/v2.5.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Grafana Mimir version 2.5 release notes"
+menuTitle: "V2.5 release notes"
+description: "Release notes for Grafana Mimir version 2.5"
+weight: 300
+---
+
+# Grafana Mimir version 2.5 release notes
+
+Grafana Labs is excited to announce version 2.5 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For the complete list of changes, see the [changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Alertmanager Discord support**
+  Alertmanager can now be configured to send alerts in Discord channels.
+
+- **Configurable TLS minimum version and cipher suites**
+  We added the flags `-server.tls-min-version` and `-server.tls-cipher-suites` that can be used to define the minimum TLS version and the supported cipher suites in all HTTP and gRPC servers in Mimir.
+
+- **Lower memory usage in store-gateway, ingester and alertmanager**
+  We made various changes related to how index lookups are performed and how the active series custom trackers are implemented, which results in better performance and lower overall memory usage in the store-gateway and ingester.
+  We also optimized the alertmanager, which results in a 50% reduction in memory usage in use cases with larger numbers of tenants.
+
+- **Improved Mimir dashboards**
+  We added two new dashboards named `Mimir / Overview resources` and `Mimir / Overview networking`. Furthermore, we have made various improvements to the following existing dashboards:
+  - `Mimir / Overview`: Add "remote read", "metadata", and "exemplar" queries.
+  - `Mimir / Writes`: Add optional row about the distributor's new forwarding feature.
+  - `Mimir / Tenants`: Add insights into the read path.
+
+### Helm chart improvements
+
+- **Zone aware replication**
+  Helm now supports deploying the ingesters and store-gateways as different availability zones. The replication is also zone-aware, therefore multiple instances of one zone can fail without any service interruption and roll outs can be performed faster because many instances of each zone can be restarted together, as opposed to them all restarting in sequence.
+
+  **This is a breaking change**, [for details on how to upgrade please review the Helm changelog](https://github.com/grafana/mimir/blob/main/operations/helm/charts/mimir-distributed/CHANGELOG.md).
+
+- **Running without root privileges**
+  All Mimir, GEM and Agent processes now don't require root privileges to run anymore.
+
+- **Unified Nginx configuration for Mimir and GEM**
+  This change allows for an easier upgrade path from Mimir to GEM, without any downtime. The unified configuration also makes it possible to autoscale the gateway pods and it supports OpenShift Route.
+
+- **Updated Minio**
+  The Minio sub-chart was updated from `4.x` to `5.0.0`, note that this update **inherits a breaking change** because the minio gateway mode was removed.
```

Comment:
```suggestion
- **Updated MinIO**
  The Minio sub-chart was updated from `4.x` to `5.0.0`, note that this update **inherits a breaking change** because the MinIO gateway mode was removed.
```

I think that's how they prefer to write it